### PR TITLE
Mvp

### PR DESCRIPTION
**Pull request by toddapetersen**
Hunter... didn't have time to squish, but here are my changes that support an MVP for Agency() including the changes we discussed regarding VIN last week and a couple other minor ones.

### CODE REVIEW

**Comment by hunterowens** [comment]
No problem @toddapetersen. I've added a bunch of comments to the doc for the MVP ideas. Might make sense to break this into smaller pull requests. 

Additionally, I have some PRs waiting on your or @LADOTBikeshare / @jannasmith / @xxmarcelxx review. See #29 #32  #35. 

I believe #32 we should merge in before we merge this. #29 might conflict, and #35 doesn't conflict. Can you provide review on that side and then I can merge and then provide some changes. 

Also, we should maybe add some sort of "nearby spots" API (see comment on spec)

File: agency/README.md [hunk 1]
```diff
@@ -27,247 +24,224 @@ The Vehicle Registration API is required in order to register a vehicle for use
...
-
-| Field | Type     | Required/Optional | Other |
-| ----- | -------- | ----------------- | ----- |
-| `trip_id` | UUID | Required | Issued by InitMovementPlan() API | 
-| `timestamp` | Unix Timestamp | Required | Time of day (UTC) data was sampled|
-| `gps_pos` | DDD.DDDDD° | Required | GPS location in decimal degrees at time of sample |
-| `park_option` | Enum | Required | Choose the type of parking place desired (`closest`, `least expensive`) |
-
-Reponse: 
+| `message` | Enum | See [Message](#message) Enum |
```
[the remark this answers is not in the document]
Comment (toddapetersen) [reply agency/README.md]: Agree.... but in a later REV.    We're going to start talking about mapping, ground truth, etc as soon as we tag.

File: agency/README.md [hunk 2]
```diff
@@ -27,247 +24,224 @@ The Vehicle Registration API is required in order to register a vehicle for use
...
+                "geometry": {
+                    "type": "Point",
+                    "coordinates": [
+                        -118.464851975441,
+                        33.990366257735
+                    ]
+                }
+            }
+        ] }
+```
```
[the remark this answers is not in the document]
Comment (toddapetersen) [reply agency/README.md]: I like the route construct for packaging multiple points.   It is handy if there are latencies, etc that warrant buffering telemetry data.    A thought...  We may want to consider distinguishing between the frequency between data points and the frequency of the live calls to the Update-Trip-Telemetry endpoint during the trip.   They could be separate.

File: agency/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (hunterowens) [reply agency/README.md]: Got it.

File: agency/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (hunterowens) [reply agency/README.md]: Yes, let's distinguish. I can work on a later PR after we merge this.

**Comment by toddapetersen** [comment]
Done.

**Pull request merged**